app/controllers/website_wizard_controller.rb

```diff
@@ -25,6 +25,8 @@ def update
         redirect_to next_wizard_path
         # render_wizard
       else
+        #notice with an active model error
+        flash[:notice] =  @website.errors.messages[:website].try(:first)
```

Comment:
This error is raised when checking for duplicate websites in our website.rb file.. forgot to tell in the above